docs/sources/flow/setup/install/binary.md

```diff
@@ -1,9 +1,13 @@
 ---
-title: Other systems
+description: Learn how to install Grafana Agent Flow on other systems
+title: Install Grafana Agent Flow on other systems
+menuTitle: Other systems
 weight: 600
+aliases:
+ - /docs/sources/flow/install/binary/
```

Comment:
Heads up: none of the aliases for the old install links are working when I run `make docs`.

Reply:
They should be fixed now